keyboards/m65/config.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID       0xBABA
+#define PRODUCT_ID      0x6060
+#define MANUFACTURER    m65
+#define PRODUCT         keyboard
+
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 13
+
+#define UNUSED_PINS
+
+#define DIODE_DIRECTION COL2ROW
+#define DEBOUNCE 5
+
+#define LOCKING_SUPPORT_ENABLE
+#define LOCKING_RESYNC_ENABLE
+
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
```

Comment:
Might want to leave this out and use the default definition:
```
quantum/command.h:#    define IS_COMMAND() (get_mods() == MOD_MASK_SHIFT)
```
If you look at `tmk_core/common/action_util.c`, you'll see that `get_mods()` returns `real_mods`, but `keyboard_report->mods` is a superset of that, and includes `weak_mods` & `macro_mods` also.

Reply:
thanks a leftover from a test I will remove it.